home/src/main/resources/config/example.applicationSetup.n3

```diff
@@ -104,7 +104,9 @@
 #        vitroWebapp:modules.tripleSource.ContentTripleSource ;
 #    # May be an absolute path, or relative to the Vitro home directory.
```

Comment:
Is this the "Vitro home directory", or the TDB directory within Vitro home?

Reply:
That is an existing comment from `tdbContentTripleSource` example. I believe `hasTdbDirectory` if relative it will be a directory under Vitro home.

Reply:
If you are asking about the RDF Delta Data source name, that directory is created by the RDF Delta patch log service.